clients/venice-client/src/main/java/com/linkedin/venice/fastclient/meta/StoreMetadataFetchMode.java

```diff
@@ -0,0 +1,5 @@
+package com.linkedin.venice.fastclient.meta;
+
+public enum StoreMetadataFetchMode {
```

Comment:
Can we some comments for these enums? 

An example would be:
`THIN_CLIENT_BASED_METADATA`: using thin client inside FC to access the meta system store via routers to get a copy of the metadata information for this store
`DA_VINCI_CLIENT_BASED_METADATA`: using DaVinci client inside FC to cache the meta system store. Using DaVinci client and FC with DaVinci has a cyclic dependency issue, so only one of DaVinci or FC with DaVinci can be used.
`SERVER_BASED_METADATA`: FC directly talks with Server endpoints to get the metadata information.

Reply:
Sure